### PR TITLE
update nacos 2.4.2,seata 2.1.0,spring boot 3.2.9

### PR DESCRIPTION
**Pull request by jaimyjie**

### CODE REVIEW

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/test/java/com/alibaba/cloud/nacos/registry/MockNamingService.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import com.alibaba.nacos.api.naming.pojo.Instance;
 import com.alibaba.nacos.api.naming.pojo.ListView;
 import com.alibaba.nacos.api.naming.pojo.ServiceInfo;
+import com.alibaba.nacos.api.naming.selector.NamingSelector;
```
Comment (yuluo-yx) [inline spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/test/java/com/alibaba/cloud/nacos/registry/MockNamingService.java:28]: why import this?

**Comment by yuluo-yx** [comment]
Do you have any examples of running nacos and Seata locally? Do they function properly?

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/test/java/com/alibaba/cloud/nacos/registry/MockNamingService.java (hunk 1 above)
Comment (jaimyjie) [reply spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/test/java/com/alibaba/cloud/nacos/registry/MockNamingService.java:28]: 2.4.2新增的，需要导入，要不编译不了

**Comment by jaimyjie** [comment]
> Do you have any examples of running nacos and Seata locally? Do they function properly?

nacos有用项目里面examples跑过正常，seata没有跑过完整案例

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/test/java/com/alibaba/cloud/nacos/registry/MockNamingService.java (hunk 1 above)
Comment (yuluo-yx) [reply spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/test/java/com/alibaba/cloud/nacos/registry/MockNamingService.java:28]: got it

**Review by yuluo-yx — APPROVED** [review]
LGTM, thanks @jaimyjie

**Comment by yuluo-yx** [comment]
ptal @ruansheng8 @chickenlj

**Comment by jaimyjie** [comment]
项目里面有几个nacos容器配置文件，是否要一起升级版本？

services:
  nacos:
    image: nacos/nacos-server:2.0.1
    container_name: nacos-standalone
    environment:
      - PREFER_HOST_MODE=hostname
      - MODE=standalone
    ports:
      - "8848:8848"
      - "9848:9848"
    healthcheck:
      test: "curl --fail http://127.0.0.1:8848/nacos/v1/console/health/liveness || exit 1"
      interval: 5s

**Comment by yuluo-yx** [comment]
> 项目里面有几个nacos容器配置文件，是否要一起升级版本？
> 
> services: nacos: image: nacos/nacos-server:2.0.1 container_name: nacos-standalone environment: - PREFER_HOST_MODE=hostname - MODE=standalone ports: - "8848:8848" - "9848:9848" healthcheck: test: "curl --fail http://127.0.0.1:8848/nacos/v1/console/health/liveness || exit 1" interval: 5s

yep

**Review by ruansheng8 — APPROVED** [review]
LGTM.

**Comment by luojianjun** [comment]
包路径改变 引用类不需要修改吗

**Pull request merged**